app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -122,10 +143,15 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
     showUserAccountStatus(R.id.patients_user_status_awaitingsmsverification)
   }
 
-  fun showStatusPatientSummarySaved(patientName: String){
+  fun showStatusPatientSummarySaved(patientName: String) {
+    nameInStatusSavedText.text = patientName
+    showSummaryStatus(R.id.patient_status_summary_saved)
   }
 
-  fun showStatusPatientAppointmentSaved(patientName: String, appointmentDate: LocalDate){
+  fun showStatusPatientAppointmentSaved(patientName: String, appointmentDate: LocalDate) {
+    nameInAppointmentSavedText.text = context.getString(R.string.patient_status_summary_saved_scheduled_appointment, patientName)
+    dateInAppointmentSavedText.text = appointmentDate.format(DateTimeFormatter.ofPattern("dd-MMM-yyyy", Locale.ENGLISH))
```

Comment:
If we don't inject the formatter, we should it a property of the `PatientsScreen` and not create it again every time we need to show the notification.

Reply:
Since the formatter is not being re-used within this class, is it still necessary to make it a class property?

Reply:
Yes, because parsing a pattern is not cheap and we don't want to create a new instance every time we come back to the patients screen and need to show a notification.

FYI, in my current branch, I have already created injectable formatters, so we can leave this as is for now and change it once my branch gets merged in.